### PR TITLE
Ui/last block ui fix

### PR DESCRIPTION
**Pull request by hsurf22**
- Fix UI: Last Blocks table
- Fix UI: Last Blocks title
- Fix UI: Last Blocks page (Mobile, Desktop)
- Fix UI: Last Blocks detail page

### CODE REVIEW

File: apps/explorer/src/modules/uncles/components/UnclesTableTitle/UnclesTableTitle.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <a v-if="pageType == 'home'" class="button-link" href="/blocks"
+        ><button>{{ $t('btn.view-all') }}</button></a
+      >
+      <slot v-if="pageType != 'home'"></slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
```
Comment (aldoborrero) [inline apps/explorer/src/modules/uncles/components/UnclesTableTitle/UnclesTableTitle.vue]: Let's try to use [vue-class-componnent](https://github.com/vuejs/vue-class-component) syntax to unify with other components

File: apps/explorer/src/components/TableTitleButton/TableTitleButton.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="table-title-button"></div>
+</template>
+
+<script>
+export default {
```
Comment (aldoborrero) [inline apps/explorer/src/components/TableTitleButton/TableTitleButton.vue]: Let's try to use [vue-class-componnent](https://github.com/vuejs/vue-class-component) syntax to unify with other components

File: apps/explorer/src/modules/uncles/components/UnclesTableTitle/UnclesTableTitle.vue (hunk 1 above)
Comment (hsurf22) [reply apps/explorer/src/modules/uncles/components/UnclesTableTitle/UnclesTableTitle.vue]: I'll take a look at it.

File: apps/explorer/src/components/TableTitleButton/TableTitleButton.vue (hunk 2 above)
Comment (hsurf22) [reply apps/explorer/src/components/TableTitleButton/TableTitleButton.vue]: I'll take a look at it.

**Comment by hsurf22** [comment]
> In general I think this looks good. Although I'm thinking it might be better to move the styling changes into a separate branch? It's a nice idea to replace the vuetify stuff with something of our own but it seems like this would maybe be better to develop independently and combine later?
> Also I have a couple of general questions:
> 
> * Why did you decide to use both scss and less? Is there something that can be done with less and not with scss or vice versa?
> * There seems to be a lot of duplication of existing code - is this because the eventual plan is to replace the existing components?

**Comment by hsurf22** [comment]
> In general I think this looks good. Although I'm thinking it might be better to move the styling changes into a separate branch? It's a nice idea to replace the vuetify stuff with something of our own but it seems like this would maybe be better to develop independently and combine later?

The reason I wanted this UI fixes merged now is that I want EthVM UI bugs cleaned up before the beta release, and I'm not trying to fix every single UI, but only the essential part of UI to keep the consistencies.

> Also I have a couple of general questions:
> 
> * Why did you decide to use both scss and less? Is there something that can be done with less and not with scss or vice versa?

Would you let me know where I used SCSS on our code...?

> * There seems to be a lot of duplication of existing code - is this because the eventual plan is to replace the existing components?

Yes. I know this. I created new (not really) reusable components from existing code, so there must be files with same codes in it. I will remove these duplicated files later.

**Comment by carolinaf86** [comment]
> > * Why did you decide to use both scss and less? Is there something that can be done with less and not with scss or vice versa?
> 
> Would you let me know where I used SCSS on our code...?
> 

This PR creates this file in scss: TableTitleButton.scss, and this file in less: BlockTableContent.less

**Pull request closed without merging**